hassio/src/components/supervisor-backup-content.ts

```diff
@@ -364,7 +365,9 @@ export class SupervisorBackupContent extends LitElement {
     if (folders?.length) {
       data.folders = folders;
     }
-    data.homeassistant = this.homeAssistant;
+
+    // onboarding needs at least homeassistant to restore
```

Comment:
We used to allow only full backups I think, do we accept partial ones too now?

Reply:
🤔 as far as I know it was supported already at onboarding

Reply:
What happens if `homeassistant` is not part on the backup?

Reply:
🤷 I don't know. I can try later. Hopefully an error from supervisor. It makes no sense on onboarding because it cannot setup your HA.

Reply:
Nothing happens, supervisor is restoring and we wait on core to refresh forever.